### PR TITLE
Fix logging style condition in WrenchLogger

### PR DESCRIPTION
**Pull request by Kydoimos97**
Updated the logging style condition to handle cases when running on AWS Lambda by adding a check for the `running_on_lambda` attribute. This ensures proper logging formatting is applied only when appropriate.

### CODE REVIEW

File: README.md
Comment (Kydoimos97) [inline README.md:1]: Added documentation on the logger updates.

File: requirements.txt
Comment (Kydoimos97) [inline requirements.txt:1]: removed openai dependency

File: WrenchCL/Tools/__init__.py
Comment (Kydoimos97) [inline WrenchCL/Tools/__init__.py:1]: Reworked the message as it will not be depreciated

File: tests/test_tools.py
Comment (Kydoimos97) [inline tests/test_tools.py:1]: made sure tests work with caplog capturing the new log stream correctly.

**Review by dalmad2 — APPROVED** [review]

**Pull request merged**